src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -111,7 +131,7 @@ public User getUser(String email) {
    }
 
   /**
-   * Gets messages posted by a user, or all messages if user is null.
+   * Gets messages posted to/by a user, or all messages if user is null.
    *
    * @return a list of any messages posted by the user, sorted by time descending. If user is null, returns all messages in the Datastore.
```

Comment:
Nitpick -- if the comment is changed to reflect the new setup, the return value description probably should be too